src/optionsMenu/settingsState.ts

```diff
@@ -22,6 +23,7 @@ const settingsState = createSlice({
     channelVUs: true,
     channelVUsStereo: true,
     resetTrimOnLoad: true,
+    saveShowPlanChanges: true,
```

Comment:
Hrm, I kinda want this to default to false - all our user education so far has been that it doesn't save, so this is quite a big change.

Reply:
fine :P